devs/docs/es-backports.rst

```diff
@@ -287,22 +287,22 @@ should be crossed out as well.
 - [ ] de6ac6aea6d Fix recovery stage transition with sync_id (#57754)
 - [ ] 46c8d00852d Remove nodes with read-only filesystems (#52680) (#59138)
 - [ ] 1ced3f0eb38 Extract recovery files details to its own class (#59121)
-- [ ] 0752a86fe58 Enforce higher priority for RepositoriesService ClusterStateApplier (#59040)
-- [ ] 00ed31d000f Remove IndexShardRoutingTable#primaryAsList (#59044)
-- [ ] f0dd9b4ace6 Add data stream timestamp validation via metadata field mapper (#59002)
-- [ ] c1781bc7e7a [7.x] Add include_data_streams flag for authorization (#59008)
-- [ ] d22dd437f18 Fix Two Common Zero Len Array Instantiations (#58944) (#58993)
-- [ ] dc9e364ff2d Count coordinating and primary bytes as write bytes (#58984)
+- [x] 0752a86fe58 Enforce higher priority for RepositoriesService ClusterStateApplier (#59040)
+- [x] 00ed31d000f Remove IndexShardRoutingTable#primaryAsList (#59044)
+- [s] f0dd9b4ace6 Add data stream timestamp validation via metadata field mapper (#59002)
+- [s] c1781bc7e7a [7.x] Add include_data_streams flag for authorization (#59008)
+- [s] d22dd437f18 Fix Two Common Zero Len Array Instantiations (#58944) (#58993)
+- [d] dc9e364ff2d Count coordinating and primary bytes as write bytes (#58984)
 - [ ] 1ef2cd7f1ad Add memory tracking to queued write operations (#58957)
```

Comment:
Parts of this would be interesting, but we'll need to check how this would interact with our existing accounting. A cherry-pick of the commit is probably not reasonable but maybe it would make sense to copy over some bits manually.